shell/common/rasterizer.cc

```diff
@@ -150,9 +152,20 @@ void Rasterizer::DoDraw(std::unique_ptr<flow::LayerTree> layer_tree) {
     return;
   }
 
+  PersistentCache* persistent_cache = PersistentCache::GetCacheForProcess();
+  persistent_cache->ResetIsAccessed();
+
   if (DrawToSurface(*layer_tree)) {
     last_layer_tree_ = std::move(layer_tree);
   }
+
+#if FLUTTER_RUNTIME_MODE != FLUTTER_RUNTIME_MODE_RELEASE
+  if (persistent_cache->IsDumpingSkp() && persistent_cache->IsAccessed()) {
```

Comment:
Instead of the #ifdefs, can all of this just be made conditional on `IsDumpingSkp` (which presumably will be false in release mode)

Reply:
I agree. Let's make sure all the code compiles in all configurations. LTO should get rid of the unreachable code anyway. I also followed the same pattern [in the raster cache](https://github.com/flutter/engine/pull/8145/files#diff-d94c1943874763e8d6cc9eed9f1ab7eaR263) which I will correct.